relay-ourlogs/src/ourlog.rs

```diff
@@ -104,6 +104,20 @@ pub fn otel_to_sentry_log(otel_log: OtelLog) -> Result<OurLog, Error> {
             Value::I64(0),
         )),
     );
+    attribute_data.insert(
+        "sentry.body".to_string(),
+        Annotated::new(OurLogAttribute::new(
+            OurLogAttributeType::String,
+            Value::String(body.clone()),
+        )),
```

Comment:
Sorry I before wasn't sure if this should be here or in the store, I now see that this is necessary for conversion into an EAP Item but doesn't belong to the ourlog.

I don't wanna make you chase stuff around, so feel free to leave it here, just didn't get it before.

Reply:
I think it's better if all attributes are defined at the same place, otherwise we'll risk to have duplicates.

If anything, I'd push for the normalization code to create a `TraceItem` directly and not even do that in the `produce_log` function.